packages/flutter_tools/lib/src/android/README.md

```diff
@@ -0,0 +1,90 @@
+# Flutter Tools for Android
+
+This section of the Flutter repository contains the command line developer tools
+for building Flutter applications on Android. What follows are some notes about
+updating this part of the tool.
+
+## Updating Android dependencies
+The Android dependencies that may need updating in Flutter project templates
+include the Android NDK and SDK versions, Gradle, the Kotlin Gradle Plugin,
+and the Android Gradle Plugin. The current template versions of these
+dependencies can be found in [gradle_utils.dart](gradle_utils.dart). The
+following details steps to take when (i) wanting to upgrade the template
```

Comment:
Nit: This reads awkwardly. What do you think about `Follow the guide below when ...` 

It would require more rewriting which I dont want to make you do but it could also make sense to move the parts of this sentence into headers with actions like `### When a new version of ABC is released` and `When  updating the default version of XYZ`